serveradmin/apps/models.py

```diff
@@ -13,7 +13,7 @@ class Application(models.Model):
     name = models.CharField(max_length=80, unique=True)
     app_id = models.CharField(max_length=64, unique=True, editable=False)
     auth_token = models.CharField(max_length=64, unique=True, editable=False)
-    owner = models.ForeignKey(User)
+    owner = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Is it really meant that Applications will be deleted too when an user is deleted?
I rather think the Applications should stay with an undefined owner, right?

Reply:
Applications here is just serveradmins name for Application Tokens. Further I think we never delete users anyway but deactivate them only. Finally this is just explicitly stating the default on_delete behavior, as is required by django2. Even though we don't support django2 yet, this is part of the work.